R/addUtilsFunctions.R

```diff
@@ -10,17 +10,27 @@
 #'     of vector variables. 
 #' }
 #' 
-#' @param outfile Character scalar, giving the path to which the script 
+#' @param outFile Character scalar, giving the path to which the script 
 #'     should be copied. The path is relative to the root of the active 
 #'     project. If a file with this name already exists, the function will 
 #'     ask for confirmation before overwriting it. 
+#' @param copyTests Logical scalar, defining whether to copy unit tests for 
+#'     the utility functions to \code{tests/testthat/tests-<name>.R}, 
+#'     where \code{<name>} is the base name of \code{outFile}.
```

Comment:
Would it make sense to mention here that in addition unit tests are setup using `usethis::use_testthat()`?

Reply:
Yes! I added a couple of lines for this.